include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -283,7 +293,8 @@ SECTIONS
 	}
 #endif
 
-#include <app_data_alignment.ld>
```

Comment:
I think this needs to remain.  The alignment required here will depend on the region size requirements.  For power of two, that needs to be a power of two align to make sure we don't let anything else get exposed.  During compile, this include file may or may not be generated and may or may not differ from 32.  And we can't use a Kconfig because in some cases we'll be using the app data size used rounded up to the next power of two.

Reply:
@agross-linaro , yes, I realized that, as well. This needs to be brought back.

And I think the very same is needed for every ram section we add in the linker..

Reply:
What bothers me, also, is the fact that app_data_alignment is attempting to align the start of next section, assuming that the next section is the Application Memory. But is looks to me that it might, instead, me the APP Shared Memory. I think the inclusion needs to be brought exactly before that section is defined.

Reply:
I guess we also want to get the python script and the corresponding cmake code to be removed at the same time.